### PR TITLE
tooltips added

### PR DESCRIPTION
**Pull request by mhigbyflowers**
Tooltips serving from service

### CODE REVIEW

File: public-ui/src/app/utils/tooltip.service.ts [hunk 1]
```diff
@@ -2,13 +2,17 @@ import { Injectable } from '@angular/core';
...
 @Injectable()
 export class TooltipService {
-  ageChartHelpText = 'The age at occurrence bar chart provides a binned age \n' +
-    'distribution for participants at the time the medical concept ' +
-    'being queried occurred in their records. \n' +
-    'If an individual’s record contains more than one mention of a concept, \n' +
-    'the age at occurrence is included for each mention. \n' +
-    'As a result, a participant may be counted more ' +
-    'than once in the distribution. ';
+  ehrDomains = `Electronic health records contain information about an individual’s health.
```
Comment (mhigbyflowers) [inline public-ui/src/app/utils/tooltip.service.ts:5]: tooltips now being served in service

File: public-ui/src/app/utils/tooltip.service.ts [hunk 2]
```diff
@@ -2,13 +2,17 @@ import { Injectable } from '@angular/core';
...
+  Medical concepts are similar to medical terms; they capture medical information in individual
+  records and may sometimes have values associated with them.
+  For example, “height” is a medical concept that has a measurement value (in centimeters).
+  These concepts are categorized into different domains. Domains are types of medical information.
+  There are numerous domains of data in EHR records,
+  but some of the most common domains include conditions, drug exposures, measurements and
+  procedures`;
+  ageChartHelpText = `The Age at First Occurrence in EHR bar chart captures the age at which the
```
Comment (SrushtiGangireddy) [inline public-ui/src/app/utils/tooltip.service.ts:13]: quote ?

File: public-ui/src/app/views/ehr-view/ehr-view.component.ts [hunk 3]
```diff
@@ -184,13 +184,13 @@ export class EhrViewComponent implements OnInit, OnDestroy {
   }
 
   public showToolTip(g) {
-    if (g === 'Biological Sex' || g === 'Gender Identity') {
+    if (g === 'biological sex' || g === 'gender identity') {
```
Comment (SrushtiGangireddy) [inline public-ui/src/app/views/ehr-view/ehr-view.component.ts]: This might as well go in tool tip service may be.

**Review by SrushtiGangireddy — APPROVED** [review]
Looks much cleaner.

**Pull request merged**